apps/complaints/acceptance/pages/confirmation.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
 module.exports = {
-  url: 'confirmation'
+  url: 'confirmation',
+  content: {
+    sent: 'The complaint has been sent.',
```

Comment:
exactly that. We bring in translations for conditional content, but something like this it would be better to check for the presence of an element IMO